### DISCUSSION TITLE
Ctrl-Shift-<Key> Combination doesn't work in Wezterm, but works in Kitty

### DISCUSSION THREAD

**Discussion by DUOLabs333** [wezterm/wezterm#5472] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

Kwin

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

On Kitty, keyboard shortcuts like `Ctrl+Shift+;` are sent to the application (which is different than `Ctrl+;`, and is separately sent to the application). I used this to make some Neovim shortcuts.

However, on Wezterm, `Ctrl+Shift+;` is sent as `Shift+;`, or `:`, while `Ctrl+;` is sent correctly. Is this expected; and if so, is there a way to change this behavior?

**Comment by DUOLabs333** [comment, marked as answer]
This was fixed with setting `enable_kitty_keyboard=true` in my config.

**Comment by dch-cap-rx** [comment]
@DUOLabs333:
This is what worked in my case to switch off the `Ctrl`+`Shift`+`-` and `Ctrl`+`Shift`+`+` without affecting `Ctrl`+`-` and `Ctrl`+`+`.:
```lua
config.keys = {
   {
      key = 'mapped:_',
      mods = 'CTRL|SHIFT',
      action = wezterm.action.DisableDefaultAssignment,
   },
   {
      key = 'mapped:+',
      mods = 'CTRL|SHIFT',
      action = wezterm.action.DisableDefaultAssignment,
   },
-- ...
}
```

**Reply by DUOLabs333** [reply]
In my case, the keyboard shortcuts weren't overriden by wezterm, it just wasn't passed in correctly.

**Reply by fikovnik** [reply]
I actually needed both: enable_kitty_keyboard and disable default assignment. Thank you!
